app/controllers/tracks/concepts_controller.rb

```diff
@@ -8,13 +8,11 @@ class Tracks::ConceptsController < ApplicationController
   def index
     @concept_map_data = Track::DetermineConceptMapLayout.(@track)
 
-    if current_user&.joined_track?(@track)
-      @concept_map_data[:status] = UserTrack::GenerateConceptStatusMapping.(@user_track)
-      render action: "index/joined"
-    else
-      @concept_map_data[:status] = {}
-      render action: "index/unjoined"
-    end
+    @concept_map_data[:status] = if current_user&.joined_track?(@track)
+                                   UserTrack::GenerateConceptStatusMapping.(@user_track)
+                                 else
+                                   {}
+                                 end
```

Comment:
We need to find a way to tell Rubocop to fuck off with this ridiculous indenting 🙂